src/tools/cephfs/cephfs-shell

```diff
@@ -882,6 +883,9 @@ class CephFSShell(Cmd):
         """
         Remove a specific Directory
         """
+        errors = {2: 'No such file or directory',
+                  20: 'Not a directory',
+                  39: 'Directory not empty'}
```

Comment:
I have same opinion here, better to not add many mappings of error code to error message and rather deal with problem in a single go.

Reply:
Same as #46185, removed. Thanks